tools/allowed-large-files

```diff
@@ -25,6 +25,7 @@ go.sum
 operator/bundle/manifests/*.yaml
 operator/config/crd/bases/*.yaml
 operator/config/manifests/bases/*.yaml
+operator/internal/central/values/translation/translation_test.go
```

Comment:
Ouch. Maybe instead it's time to start chipping away at this file? How about putting `TestDeploymentDefaults` into a new file such as `deployment_test.go`? WDYT?